src/json-formatter.less

```diff
@@ -76,6 +76,16 @@
       transform: rotate(90deg);
     }
   }
+
+  >a>.thumbnail-text {
```

Comment:
I think there must use ">". Preview show in hover(:hover). Space selector may cause all descendant elements influence by style.

example:
Selector ```object:hover value```

```
<object>
  <value></value> // <----- I will show this
  <children>
    <object>
      <value></value> // <----- But he also show
      <children>
      </children>
    </object>
  </children>
</object>
```

Selector ```object:hover>value```  can be done